autometa/common/markers.py

```diff
@@ -201,21 +201,22 @@ def get(self, format='wide', **kwargs):
                 orfs=self.orfs_fpath)
         return Markers.load(fpath=self.markers_fp, format=format)
 
-def main(args):
-    markers = Markers(orfs_fpath=args.orfs, kingdom=args.kingdom, dbdir=args.dbdir)
-    markers.get()
-
-if __name__ == '__main__':
+def main():
     import argparse
     import logging as logger
     logger.basicConfig(
         format='%(asctime)s : %(name)s : %(levelname)s : %(message)s',
         datefmt='%m/%d/%Y %I:%M:%S %p',
         level=logger.DEBUG)
-    parser = argparse.ArgumentParser('')
+    parser = argparse.ArgumentParser(description='Annotate ORFs with kingdom-marker information')
     parser.add_argument('orfs', help='</path/to/prot.orfs.faa>')
```

Comment:
As this script may be run standalone by the user, the help messages have to be very clear. Here, you should explain what type of file you want, so "Path to a fasta file containing amino acid sequences of open reading frames".